wren-ai-service/eval/prediction.py

```
@@ -138,12 +142,12 @@ def init_providers(mdl: dict) -> dict:
     if engine_config is None:
         raise ValueError("Invalid datasource")
 
-    providers = provider.init_providers(engine_config=engine_config)
+    providers_inner = provider.init_providers(engine_config=engine_config)
```

Ruff (0.8.2)

Undefined name `provider`

(F821)